routingpy/routers/valhalla.py

```diff
@@ -569,17 +663,18 @@ def _parse_matrix_json(response, units):
 
         return Matrix(durations=durations, distances=distances, raw=response)
 
-    def _build_locations(self, coordinates):
+    @staticmethod
+    def _build_locations(coordinates):
         """Build the locations object for all methods"""
 
         locations = []
 
         # Isochrones only support one coordinate tuple, so check for type of first element
-        if isinstance(coordinates[0], (list, tuple, self.Waypoint)):
+        if isinstance(coordinates[0], (list, tuple, Valhalla.Waypoint)):
             for idx, coord in enumerate(coordinates):
                 if isinstance(coord, (list, tuple)):
                     locations.append({"lon": coord[0], "lat": coord[1]}),
-                elif isinstance(coord, self.Waypoint):
+                elif isinstance(coord, Valhalla.Waypoint):
                     locations.append(coord._make_waypoint())
```

Comment:
same for this method

Reply:
here I'm not sure which method you're referring to, but I think those `get_params` methods were the only ones which are supposed to be used by consumers